snap/snapcraft.yaml

```diff
@@ -0,0 +1,63 @@
+name: plume
+base: core18
+version: '0.3.0' # just for humans, typically '1.2+git' or '1.3.2'
+summary: Multi-user blogging platform, federated over ActivityPub
+description: |
+  There is normally some text here; it should probably not be
```

Comment:
Maybe you are already aware of that, but just to have reminder: this description should probably be updated.

Reply:
Updated. Please give the verbiage a once-over!